### DISCUSSION TITLE
'Console only' option missing in 4.0.9 and 4.1.0

### DISCUSSION THREAD

**Discussion by Eagle-Admin** [utmapp/UTM#4593] [UTM Help (macOS)]
**Describe the issue**  
'Console only' option is missing in recent releases. It's supposed to be a dropdown menu in the Display tab when editing the VM.

It is described here: https://www.kali.org/docs/virtualization/install-utm-guest-vm/
As well as hinted in the official guide for installing Kali on M1: https://docs.getutm.app/guides/kali/

**Configuration**  
* UTM Version: 4.1.0 (and 4.0.9)
* macOS Version: 13.0
* Mac Chip: M1 Pro

**Steps to reproduce**
Create new VM > Virtualize
Choose ISO
Edit VM
Display > no option for 'Console only'.

**VM** 
Kali Linux ARM64 offline installer.

**Solution**
Create new VM > Virtualize
Choose ISO
Edit VM
Add device > Serial
Click save.
Boot VM - two windows will appear now, click in the new terminal/serial window and complete the installation from this.
<img width="705" alt="Screenshot 2022-11-03 at 09 11 53" src="https://user-images.githubusercontent.com/117160375/199673195-d1e3907f-7dd3-47bf-a40a-e77abf36e2ca.png">

<img width="1392" alt="Screenshot 2022-11-03 at 09 15 13" src="https://user-images.githubusercontent.com/117160375/199673725-7d71f3bf-3eb4-45b8-9a19-1e0a889d85c4.png">

**Comment by AzhariRamadhan** [comment]
same issue

**Comment by AzhariRamadhan** [comment]
u can try this , https://www.youtube.com/watch?v=M_IxKfeqbx8

**Reply by Eagle-Admin** [reply]
Thank you! I was able to make the installer work now.

**Comment by osy** [comment]
https://docs.getutm.app/guides/kali/#black-screen-on-start

**Reply by Eagle-Admin** [reply]
** Updated ** 

I must have read a guide somewhere else and mistaken it for the official guide.

**Reply by osy** [reply]
At what point is “console only” mentioned? Did you click the right link?

**Reply by Eagle-Admin** [reply]
Okay that's super weird. I could swear nothing was written about this serial in the official guide 2 days ago??? I can see the commit is 15 days ago, but was the page recently updated?

I dismissed your link immediately as I had already looked at it 2 days ago.

**Reply by osy** [reply]
No it was there when I posted it that’s why I asked if you clicked the right link.

**Reply by Eagle-Admin** [reply]
I must eat my own hat, bite the dust, and extend my humblest apologies.
